evap/staff/templates/staff_user_import.html

```diff
@@ -33,13 +33,31 @@ <h3>{% trans 'Import users' %}</h3>
                     <button name="operation" value="test" type="submit" class="btn btn-primary form-submit-btn">{% trans 'Upload and test' %}</button>
                 {% else %}
                     <button name="operation" value="test" type="submit" class="btn btn-light form-submit-btn">{% trans 'Upload and test' %}</button>
-                    <button name="operation" value="import" type="submit" class="btn btn-primary form-submit-btn">{% trans 'Import previously uploaded file' %}</button>
+                    <button name="operation" value="import" type="button" onclick="importUserModalShow('import', 'users');" class="btn btn-primary form-submit-btn">{% trans 'Import previously uploaded file' %}</button>
                 {% endif %}
             </div>
         </div>
     </form>
 {% endblock %}
 
+{% block modals %}
+{{ block.super }}
+    {% trans 'Import Users' as title %}
+    {% blocktrans asvar question %}Do you really want to import <strong id="data-label"></strong> from the Excel file? {% endblocktrans %}
```

Comment:
you don't need to include the data-label here and in the other new modals. they are needed for the deletion confirmation of individual entries to make sure the user deletes the correct item, but here we don't have to fear confusion because there are not many entries that all look alike.